src/clj_gatling/pipeline.clj

```diff
@@ -92,7 +92,7 @@
                    context] :as options}]
   (let [users-by-node (if rate
                         (split-equally nodes (range (max-users rate timeout-in-ms)))
-                        (split-equally nodes (range concurrency)))
+                        (split-equally nodes (range (* concurrency 2))))
```

Comment:
This gives us some 'spare' users when running a concurrent scenario, so that if the pre or post hooks are heavyweight, we have some additional users that should be able to run to maintain the desired concurrency.